tracecat/db/engine.py

```diff
@@ -157,7 +191,7 @@ def get_session_context_manager() -> contextlib.AbstractContextManager[Session]:
     return contextlib.contextmanager(get_session)()
 
 
-def get_async_session_context_manager() -> (
+async def get_async_session_context_manager() -> (
```

Comment:
this looks wrong as we are creating an async context manager as the output of the function, not invoking it as a coroutine
```suggestion
def get_async_session_context_manager() -> (
```